docs/_shared/note-local-dtrain-job.txt

```diff
@@ -0,0 +1,6 @@
+.. note::
+
+   This tutorial provides instructions for running a local distributed training job. Your setup may
+   be different. For example, for instructions on how to run a remote distributed training job,
+   visit the :ref:`qs-mdldev`.
```

Comment:
I'm a bit confused about this, because I don't think the example in this code is distributed.

Can you elaborate a bit on what you're trying to communicate in this note?

Reply:
thanks for catching this. i've removed it.